components/library/botan/Makefile

```diff
@@ -13,58 +13,61 @@
 # Copyright 2021 Andreas Wacknitz
 #
 
-BUILD_BITS= 64_and_32
-#USE_OPENSSL11= yes
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		botan
-COMPONENT_VERSION=	1.10.17
-COMPONENT_LICENSE=	BSD
+COMPONENT_VERSION=	2.19.2
+COMPONENT_LICENSE=	BSD 2-Clause "Simplified" License
 COMPONENT_PROJECT_URL=	https://botan.randombit.net/
 COMPONENT_SUMMARY= 	Crypto and TLS for C++11
 COMPONENT_SRC=		Botan-$(COMPONENT_VERSION)
-COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tgz
-COMPONENT_ARCHIVE_HASH=	sha256:6847ffb64b8d2f939dccfecc17bd2c80385d08f7621e2c56d3a335118e823613
-COMPONENT_ARCHIVE_URL=	https://botan.randombit.net/releases/old/$(COMPONENT_ARCHIVE)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH=	\
+	sha256:3af5f17615c6b5cd8b832d269fb6cb4d54ec64f9eb09ddbf1add5093941b4d75
+COMPONENT_ARCHIVE_URL=	https://botan.randombit.net/releases/$(COMPONENT_ARCHIVE)
+COMPONENT_FMRI=library/security/botan
+COMPONENT_CLASSIFICATION=org.opensolaris.category.2008:System/Security
```

Comment:
The `org.opensolaris.category.2008:` part should not be there.  See `grep COMPONENT_CLASSIFICATION make-rules/ips.mk`.